content_scripts/hints.js

```diff
@@ -384,7 +468,10 @@ var Hints = (function(mode) {
     }
 
     function createHints(cssSelector, attrs) {
-        return (cssSelector.constructor.name === "RegExp") ? createHintsForTextNode(cssSelector, attrs) : createHintsForClick(cssSelector, attrs);
+        if (cssSelector.constructor.name === "RegExp") {
```

Comment:
Personally I always try to avoid multiple `return` in one function. I see your reason to make this change to make this line shorter and more readable for human. Then how about

    return (cssSelector.constructor.name === "RegExp") ?
        createHintsForTextNode(cssSelector, attrs) : createHintsForClick(cssSelector, attrs);